src/screens/AmountInput.tsx

```diff
@@ -10,38 +10,35 @@ import NumberInput from "@/components/inputs/NumberInput";
 import Heading from "@/components/typography/Heading";
 import Label from "@/components/typography/Label";
 import { spacing } from "@/constants/DesignTokens";
-import { withdrawAtom } from "@/lib/atoms";
+import { amountInputAtom } from "@/lib/atoms";
 
 type WithdrawAmountProps = {
-  onNext: () => void;
+  title: string;
+  subTitle: string;
+  disabledText: string;
+  onNext: (amount: string) => void;
 };
 
-export default function WithdrawAmount({ onNext }: WithdrawAmountProps) {
+export default function AmountInput({
+  title,
+  subTitle,
+  disabledText,
+  onNext
+}: WithdrawAmountProps) {
```

Comment:
This type should match the name of the component followed by `Props`

```suggestion
}: AmountInputProps) {
```